packages/plans-grid/src/plans-interval-toggle/index.tsx

```diff
@@ -0,0 +1,95 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { useI18n } from '@automattic/react-i18n';
+import { sprintf } from '@wordpress/i18n';
+import { Popover } from '@wordpress/components';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import SegmentedControl from '../segmented-control';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export type BillingIntervalType = 'yearly' | 'monthly';
```

Comment:
After #48790, this type can be removed and we can import that type from the data-store instead